File: 01 - Automata.Presentation/Automata.UI/src/app/entities/project/project.ts
export class Project
{
    private _id : number;
    private _name: string;
    private _url: string;

    get Id(): number { return this._id; }
    set Id(value: number) { this._id = value; }

    get Name(): string { return this._name; }
    set Name(value: string) { this._name = value; }

    get Url(): string { return this._url; }
    set Url(value: string) { this._url = value; }

    constructor (id?: number, name?: string, url?: string) {
        this._id = id;
        this._name = name;
        this._url = url;
    }
}

File: 02 - Automata.Logic/Automata.WebApi/Entities/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Automata.WebApi.Entities
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }

        public Project() { }

        public Project(int id, string name, string url)
        {
            this.Id = id;
            this.Name = name;
            this.Url = url;
        }
    }
}